.eslintrc

```diff
@@ -9,11 +9,20 @@
     "react/forbid-prop-types": [1, { "forbid": ['object', 'any'] }],
     "class-methods-use-this" : 0,
     "semi": ["error", "never"],
+    "max-len": ['error', 120, 2, {
+      ignoreUrls: true,
+      ignoreComments: false,
+      ignoreRegExpLiterals: true,
+      ignoreStrings: true,
+      ignoreTemplateLiterals: true,
```

Comment:
Perhaps. I basically copied this from the airbnb preset and changed the number to 120. Making the rule look only at the line number introduces more linter errors.